blog.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+     <meta charset="utf-8">
+     <meta name="viewport" content="width=device-width, initial-scale=1">
+     <title>Adoptadog: Obtain a New Furry Friend Today!</title>
+     <link href="https://fonts.googleapis.com/css?family=Assistant&display=swap" rel="stylesheet">
+     <link href="https://fonts.googleapis.com/css?family=Eczar&display=swap" rel="stylesheet">
+     <link rel="stylesheet" type="text/css" href="styles/styles.css">
+  </head>
+  <body>
+      <p><a href="#skip-navigation" class="skip-hidden">Skip navigation links</a></p>
+    <header>
+        <div class="logo-container">
+          <img src="images/adoptadog-logo.svg" height="100" alt="business logo with sideview of dog in white">
+          <h1>Seattle's premiere dog adoption agency</h1>
+        </div>
+        <div class="cart-price">
+          <div class="cart">
+            <img src="images/cart-icon.svg" height="21" alt="icon of white cart">
+          </div>
+            <button class="checkout">Checkout</button>
+        </div>
+    </header>
+    <nav>
+      <ul class="navigation">
+        <li id="tab1" class="nav"><a href="index.html">Home</a></li>
+        <li class="nav"><a href="dogs.html">Dogs</a></li>
+        <li class="nav"><a href="blog.html">Blog</a></li>
+        <li class="nav"><a href="contact.html">Contact</a></li>
+      </ul>
+    </nav>
+    <section>
+      <h2>Adotpadog Blog</h2>
+        <div class="blog-body">
+          <img src="images/blog-1.jpg" alt="person sitting at the edge of a canyon with a dog, looking at the view" height="220">
+          <div class="blog-paragraph">
+            <h3>Traveling With Your Dog</h3>
+            <p>Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem
+            quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium
+            hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium
+            faccull oribus es quidignis ipietus explam sus am aut amet ant fugiatum, utem non reptat.</p>
+            <p>Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con
+            et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad
+            qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam
+            voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit
+            ulparume nonseca estorer spernam.</p>
+          </div>
+        </div>
+        <div>
+          <div class="blog-body">
+            <img src="images/blog-2.jpg" alt="several dogs on a leash, waiting to follow their leader" height="220">
```

Comment:
for now these inline height elements are ok, but when you fold in your css file you'll want to handle all the styling from the stylesheet.